app.xmcl.voxelum.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: app.xmcl.voxelum
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: xmcl-wrapper.sh
+build-options:
+  no-debuginfo: true
+finish-args:
+  # Basic System Access
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
```

Comment:
`home` needs an exception but first you should justify why you need it and why it can't be specific directories like `~/.minecraft` etc.